src/app/components/AlertMessage/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import i18next from 'i18next';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  message: string;
+  type: 'error' | 'success' | 'warning';
+}
+
+function AlertMessage({ message, type }: Props) {
+  const getTitle = () => {
+    switch (type) {
+      case 'error':
+        return i18next.t('AlertMessage:titleError');
+      case 'success':
+        return i18next.t('AlertMessage:titleSuccess');
+      case 'warning':
+        return i18next.t('AlertMessage:titleWarning');
+      default:
+        return '';
+    }
+  };
+
+  return (
+    <div className={`${styles.alertMessage} ${styles[type]}`} role="alert">
+      <div className={styles.title}>{getTitle()}</div>
```

Comment:
this should not be a div, should be a text tag like a <span> or <h4|h5|h6> or <p>, as a suggestion add it a `role="alert"` for future tests